docs/source/tutorial/first_steps.rst

```diff
@@ -186,3 +186,32 @@ executed with one of the previous examples.
     ...     ],
     ... )
     >>> # print(results)
+
+
+Training Callbacks
```

Comment:
@cthoyt this may be a bit minimalistic; also I am not sure whether this is the best place

Reply:
307521e  +  54711ae